src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -85,23 +85,23 @@
     @# FIXME: Convert argv from strings to the correct types.
     parser = argparse.ArgumentParser()
     @join param : sample.initCode.argDefaultParams
-      parser.add_argument('--{@param.identifier}')
+      parser.add_argument('--{@param.identifier}', default='')
     @end
     args = parser.parse_args()
 
-    sample_{@apiMethod.name}({@sampleFuncArgs(sample.initCode.argDefaultParams)})
+    sample_{@apiMethod.name}({@actualArgs(sample.initCode.argDefaultParams)})
 
   if __name__ == '__main__':
     main()
 @end
 
-@private sampleFuncArgs(defaultParams)
+@private actualArgs(defaultParams)
   @join param : defaultParams on ", "
     args.{@param.identifier}
   @end
 @end
 
-@private sampleFuncParams(defaultParams)
+@private formalArgs(defaultParams)
```

Comment:
Here, too, I suggest s/`defaultParams`/`sampleArgs`/ for clarity

Reply:
Done